### PR TITLE
Create Site page placeholders and minor changes done

### PR DESCRIPTION
**Pull request by anuj1511**

### CODE REVIEW

File: pages/landing/particularSite.js [hunk 1]
```diff
@@ -12,9 +12,7 @@ import { useRouter } from "next/router";
 import axios from "axios";
 
 export default function ParticularSiteComponent() {
-
-
-  const router = useRouter()
+  const router = useRouter();
   console.log(router.query.id);
```
Comment (rishabhborn2win) [inline pages/landing/particularSite.js]: remove these unwanted console.logs

File: pages/landing/particularSite.js [hunk 2]
```diff
@@ -24,9 +22,9 @@ export default function ParticularSiteComponent() {
   }, []);
 
   const getDetails = async () => {
-    if(state.userInfo?.token){
+    if (state.userInfo?.token) {
       closeSnackbar();
-      enqueueSnackbar("Signin", {varient: "success"});
+      enqueueSnackbar("Signin", { varient: "success" });
```
Comment (rishabhborn2win) [inline pages/landing/particularSite.js]: Not Relevant remove this line

**Review by rishabhborn2win — CHANGES_REQUESTED** [review]

File: pages/landing/particularSite.js (hunk 1 above)
Comment (anuj1511) [reply pages/landing/particularSite.js]: Removed

File: pages/landing/particularSite.js (hunk 2 above)
Comment (anuj1511) [reply pages/landing/particularSite.js]: Done sir

**Comment by anuj1511** [comment]
Completed the suggested changes sir

**Review by rishabhborn2win — APPROVED** [review]

**Pull request merged**